src/main/components/RequestBar.jsx

```diff
@@ -16,12 +16,33 @@ const RequestBar = (props) => {
 
   function sendFetch(e) {
     e.preventDefault();
-    const sendingObj = { method: selected, mode: 'cors' };
-    if (SourceOrDest === 'dest') sendingObj.body = JSON.stringify(requestBody);
 
-    fetch(uri, sendingObj)
-      .then(res => res.json())
-      .then(res => setData(res));
+    if (SourceOrDest === 'source') {
+      const sendingObj = { method: selected, mode: 'cors' };
+
+      fetch(uri, sendingObj)
+        .then(res => res.json())
+        .then(res => setData(res));
+    }
+    else if (SourceOrDest === 'dest') {
+      let testsClone = [...tests];
+      const sendingObj = { method: selected, mode: 'cors' };
+      let counter = 0;
+      for (let i = 0; i < testsClone.length; i++) {
+        sendingObj.body = JSON.stringify(testsClone[i].payload);
+  
+        fetch(uri, sendingObj)
+          .then(response => {
```

Comment:
lets add curly braces around the response since it is not an implicit return and the functions contains curly braces.

Reply:
I fixed this manually, was blocking other features